doc/radosgw/admin.rst

```diff
@@ -827,53 +827,63 @@ all unauthenticated users:
 Usage
 =====
 
-The Ceph Object Gateway logs usage for each user. You can track
-user usage within date ranges too.
+The Ceph Object Gateway logs the usage of each user. You can track the usage of
+each user within date ranges.
 
-- Add ``rgw_enable_usage_log = true`` in [client.rgw] section of ceph.conf and restart the radosgw service. 
+- Add ``rgw_enable_usage_log = true`` in the ``[client.rgw]`` section of
```

Comment:
Until we get a linkable macro to handle the various ways of setting options, suggest we just say to set this true in central config or Ceph.conf and restart all RGWs

Reply:
Accepted.